packages/components/psammead-episode-list/src/Image.jsx

```diff
@@ -24,19 +24,14 @@ const Wrapper = styled.div`
   display: inline-block;
   position: relative;
   width: 70px;
-  margin: 0 ${GEL_SPACING} 0 0;
-  :dir(rtl) {
-    margin: 0 0 0 ${GEL_SPACING};
-  }
+  ${({ dir }) => `margin-${dir === 'ltr' ? 'right' : 'left'}: ${GEL_SPACING};`}
```

Comment:
Just wondering why this needed changed - the `:dir` selector is used in some other components too

Reply:
@ryanmccombe it wasn't working for me. You can this here https://bbc.github.io/psammead/?path=/story/components-episodelist--video-with-surrounding-components-dark

Reply:
you'll need to change the service to Arabic

Reply:
Oh wow, it is just a draft proposal and only implemented in firefox - https://developer.mozilla.org/en-US/docs/Web/CSS/:dir